osp-deploy/cinder/powerflex/README.md

```diff
@@ -152,3 +156,69 @@ Install the Storage Data Client (SDC) on all nodes after deploying the overcloud
 ### Testing the configured Backend
 Finally create a volume-type and test if you can successfully create and attach volumes of that type.
 
+## Upgrading from RHOSP13 to RHOSP16.1
+
+### Upgrading PowerFlex
+
+First of all, upgrade PowerFlex system using official instructions. Upgrading SDC is not necessary as it has to be upgraded after Overcloud nodes OS upgrade.
```

Comment:
suggest:
1. state that before you upgrade RHOSP from 13 to 16, upgrade powerflex
2. specify the minimal version of powerflex we recommend, and maybe pointer to pwoerflex documentation on what version of powerflex is compatible with RHEL versions.
3. you will need to upgrade all parts of powerflex including SDS running on storage nodes or compute nodes (for HCI), SDC on controllers and compute nodes. [ add whatever nodes used for MDS and gateway]

Reply:
3. PowerFlex internal tool upgrades all necessary components itself without downtime. You just need to upload all necessary packages.